wiki/en/Software-Manual.md

```diff
@@ -52,30 +52,30 @@ This manual documents the Jamulus client application for use by musicians and si
 
 ## Input
 
-This shows the level of the two stereo channels for your audio input.
-Make sure not to clip the input signal to avoid distortions of the audio signal (the LEDs will indicate clipping when it occurs).
+Shows the level of the two stereo channels for your audio input.
+Make sure not to clip the input signal to avoid distortions of your sound (the LEDs will indicate clipping when it occurs).
 
 ## Mute Myself button
 
 Cuts your audio stream to the server so that you will be able to hear yourself and see your own input levels, but other musicians will not. Be aware that other musicians will not know if you have muted yourself.
 
 ## Reverb effect
 
-Reverb can be applied to one local mono audio channel or to both channels in stereo mode.
+Adds reverb to your local mono audio channel, or to both channels in stereo mode.
 The mono channel selection and the reverberation level can be modified. For example, if a microphone signal is fed
 in to the right audio channel of the sound card and a reverb effect needs to be applied, set the channel selector
 to the right and move the fader upwards until the desired reverb level is reached.
 
 
 ## Chat
 
-The chat text entered in that dialog is sent to
-all connected clients. If a new chat message arrives and the Chat dialog is not already open, it will
+Opens the chat window. Text entered is sent to
+all connected clients. If a new chat message arrives and the Chat dialoge is not already open, it will
```

Comment:
"Dialog" or "dialogue". I think I recall discussing this term but can't remember the conculsion. "Dialog" was common in IT contexts but "dialogue" is more correct in strict adherence to GB English.